packages/app/src/files-manager/actions/continue-pending-upload.action.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { DeviceType } from "App/device/constants"
+import { createAsyncThunk } from "@reduxjs/toolkit"
+import { State } from "App/core/constants/state.constant"
+import { ReduxRootState } from "App/__deprecated__/renderer/store"
+import { FilesManagerEvent, DeviceDirectory } from "App/files-manager/constants"
+import { uploadFilesRequest } from "App/files-manager/requests"
+import { getFiles } from "App/files-manager/actions/get-files.action"
+import {
+  setPendingFilesToUpload,
+  setUploadBlocked,
+  setUploadingFileCount,
+  setUploadingState,
+} from "App/files-manager/actions/base.action"
+import { loadStorageInfoAction } from "App/device/actions/load-storage-info.action"
+import { getFreeFilesSlotsCountForHarmony } from "App/files-manager/helpers/get-free-files-slots-count-for-harmony.helper"
+
+export const continuePendingUpload = createAsyncThunk(
```

Comment:
what is the difference between this action and `uploadFile`? 🤔 You can't use one of them? 🤔

Reply:
The topic has been discussed and will be resolved in the next iteration.